delphi/translators/GrFN2WiringDiagram/example_GrFN2WD_translation.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
Similar comment as about the other notebook - please clear the outputs of the notebook and then recommit it.